src/scripts/main.js

```diff
@@ -1,7 +1,272 @@
 'use strict';
+class Game {
+  constructor(initialState) {
+    this.board = initialState || [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ];
+    this.score = 0;
+    this.status = 'idle';
+  }
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+  moveLeft() {
+    let moved = false;
 
-// Write your code here
+    for (let i = 0; i < 4; i++) {
```

Comment:
Optional: The logic you have written in every `move` method is similar. have you tried to re-use this logic? If not - try to do it

Reply:
<img width="599" alt="image" src="https://github.com/mate-academy/js_2048_game/assets/70220667/3212ed9d-7313-41df-9289-49be91f107f7">
try to fix it, it will simplify the logic